Library/Formula/libdecnumber.rb

```diff
@@ -0,0 +1,41 @@
+class Libdecnumber < Formula
+  desc "ANSI C implementation of General Decimal Arithmetic"
+  homepage "http://speleotrove.com/decimal/#decNumber"
+  url "http://download.icu-project.org/files/decNumber/decNumber-icu-368.zip"
+  version "3.68"
+  sha256 "14ec2cf30b58758493a7661b78b80abfb281652b61a425b85cda83173518fe25"
+  resource "cmakelists" do
```

Comment:
The CMakeList is being used to build static and shared libraries. It is also used to install the libraries and also the headers. I could install the headers with include.install but that would be useless without the static and shared libraries.